server/db/cardData.js

```diff
@@ -0,0 +1,21 @@
+const connection = require('./')
+
+// function getCards (db = connection) {
+//   return db('ratings')
+// }
+
+function addDate (data, db = connection) {
+  return db('dates')
+    .insert(data)
+    .returning('id')
```

Comment:
As you've probably already discovered, this won't work with SQLite3 (although it'll be fine with Postgres).

Reply:
The `.returning`, that is :smile: